src/components/vm/consoles/vnc.jsx

```diff
@@ -68,6 +69,18 @@ const Enum = {
     KEY_DELETE: 111,
 };
 
+export class VncState extends ConsoleState {
+    constructor() {
+        super();
+        this.sizeMode = "none";
```

Comment:
Can we type this to be a enum-like, i.e.`"none" | "local" | "remote"` ? I suppose that first requires TS-ifying the whole new console/VNC code that just landed, so not blocking this PR.